acme4j-client/src/main/java/org/shredzone/acme4j/impl/AbstractAcmeClient.java

```diff
@@ -28,10 +28,7 @@
 import org.jose4j.jwk.PublicJsonWebKey;
 import org.jose4j.jws.JsonWebSignature;
 import org.jose4j.lang.JoseException;
-import org.shredzone.acme4j.AcmeClient;
-import org.shredzone.acme4j.Authorization;
-import org.shredzone.acme4j.Registration;
-import org.shredzone.acme4j.Status;
+import org.shredzone.acme4j.*;
```

Comment:
Please do not use import *, but the concrete classes, to keep the code style consistent.